src/types/project_manifest.rs

```diff
@@ -21,6 +21,7 @@ pub struct ProjectConfigFile {
     requirements: Option<Value>,
     analysis: Option<Vec<String>>,
     costs_version: Option<u32>,
+    parser_version: Option<u32>
```

Comment:
What's your take on spawning a new section in the schema of the manifest, dedicated to tweaking the VM / REPL internals?

Reply:
Yes, that's probably a good idea. My check-checker changes (PR coming soon) is also tweaking this structure, to pass options directly to analysis passes:
```toml
[project.analysis_settings.check_checker]
trusted_sender = true
```

Do you think something like this would make sense:
```toml
[project]
name = "my-contract"
authors = []
description = ""
telemetry = false
requirements = []

[contracts.nft-trait]
path = "contracts/nft-trait.clar"
depends_on = []

[contracts.test]
path = "contracts/test.clar"
depends_on = ["nft-trait"]

[repl]
parser_version = 1
analysis = ["check_checker"]

[repl.analysis_settings.check_checker]
trusted_sender = true
```

Reply:
Now I'm thinking the `analysis_settings` struct is not necessary and it could just be `repl.check_checker`.

Reply:
Ok, actually I have a much nicer fix to this in my check-checker branch. Mind if we hold off on this change until that one?

Reply:
This is implemented in the latest commits.